classes/classes/Scenes/Monsters/pregnancies/PlayerBasiliskPregnancy.as

```diff
@@ -0,0 +1,135 @@
+package classes.Scenes.Monsters.pregnancies 
+{
+	import classes.GlobalFlags.kGAMECLASS;
+	import classes.Player;
+	import classes.PregnancyStore;
+	import classes.Scenes.PregnancyProgression;
+	import classes.Scenes.VaginalPregnancy;
+	import classes.internals.GuiOutput;
+	
+	/**
+	 * Contains pregnancy progression and birth scenes for a Player impregnated by Basilisk.
+	 */
+	public class PlayerBasiliskPregnancy implements VaginalPregnancy
+	{
+		private var output:GuiOutput;
+		private var pregnancyProgression:PregnancyProgression;
+		
+		/**
+		 * Create a new Basilisk pregnancy for the player. Registers pregnancy for Basilisk.
+		 * @param	pregnancyProgression instance used for registering pregnancy scenes
+		 * @param	output instance for gui output
+		 */
+		public function PlayerBasiliskPregnancy(pregnancyProgression:PregnancyProgression, output:GuiOutput) 
+		{
+			// needed as an instance variable for refactor test code (detectVaginalBirth)
+			this.pregnancyProgression = pregnancyProgression;
+			this.output = output;
```

Comment:
I'm not a fan of passing references around the codebase through constructors, like this.

As an example I've set up `Output` as a singleton, so
```as3
output = Output.init();
```
Should do without the need of the `output` param.

For the first param maybe something, like
```as3
		public function PlayerBasiliskPregnancy(pregnancyProgression:PregnancyProgression = null) 
		{
			// needed as an instance variable for refactor test code (detectVaginalBirth)
			this.pregnancyProgression = pregnancyProgression || PregnancyProgression.init();
```
Or something like that could work.